### PR TITLE
Updated the OCD IDs for Paraguay, so now we also have the departments

### PR DESCRIPTION
**Pull request by marisviana**

### CODE REVIEW

**Comment by marisviana** [comment]
@jloutsenhizer Hey Justin, could you please take a look at the proposed changes for Paraguay? Thanks!

File: identifiers/country-py/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This folder lists the following [divisions of Paraguay](https://en.wikipedia.org/wiki/Departments_of_Paraguay):
```
Comment (jloutsenhizer) [inline identifiers/country-py/README.md:1]: Consider adding a note that the identifiers for `capital` and `department` use iso 3166-2 codes

**Review by jloutsenhizer — CHANGES_REQUESTED** [review]
Overall the set of entities look correct to me. A couple small changes requested.

File: identifiers/country-py/departments.csv [hunk 2]
```diff
@@ -0,0 +1,19 @@
+id,name
+ocd-division/country:py/capital:py-asu,Asunción
```
[the remark this answers is not in the document]
Comment (marisviana) [reply identifiers/country-py/departments.csv]: Just removed that

File: identifiers/country-py/README.md (hunk 1 above)
Comment (marisviana) [reply identifiers/country-py/README.md:1]: Just added this info

File: identifiers/country-py/departments.csv (hunk 2 above)
[the remark this answers is not in the document]
Comment (jloutsenhizer) [reply identifiers/country-py/departments.csv]: Sorry I don't think I was clear before. This prefix should be removed from the `district` OCD IDs as well

**Review by jloutsenhizer — APPROVED** [review]

**Comment by marisviana** [comment]
Thanks Justin! @jpmckinney, could you please take a look at these changes and see if they make sense? Thanks!

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**